Dockerfile

```diff
@@ -56,7 +56,7 @@ ENV CRATE_HEAP_SIZE 512M
 # if we cannot get any response from the CrateDB (connection refused, timeout
 # etc). If any response is received (regardless of http status code) we
 # consider the node as running.
-HEALTHCHECK --timeout=30s --interval=30s CMD curl --fail --max-time 25 $(hostname):4200
+HEALTHCHECK --timeout=30s --interval=30s CMD curl --max-time 25 $(hostname):4200 || exit 1
```

Comment:
It's not needed, no - it's just so that we either get `0` or `1` as error codes, and not any error codes that are specific to curl (like `7` for a connection error). It's a common pattern in HEALTHCHECKs, I believe.